readme.md

```diff
@@ -17,6 +17,13 @@ Install `yo`, `grunt-cli`, `bower`, `generator-angular` and `generator-karma`:
 npm install -g grunt-cli bower yo generator-karma generator-angular
 ```
 
+Install Ruby (required in order to install compass which is used by this generator)
```

Comment:
re-worded but hopefully soon removed since ruby-sass will be replaced with node-sass